### ISSUE TITLE
How to compress video manually

### ISSUE THREAD

**Issue by haibert** [numandev1/react-native-compressor#107]
### Current behavior
Does anyone have any code example of a manual compression of a video? I cant figure out what to put for the bitrate for videos from the documentation and it makes the app crash. 

### Expected behavior
I just wish the documentation showed a better example of the manual video compression, its just showing an empty object as the compression options.

### Platform
- [] Android
- [X] iOS

### React Native Version
63.4

### React Native Compressor Version
1.5.2

**Comment by pierroo** [comment]
Did you find any answer?
For example the bitrate, it says "number" and that's it, but no idea what it should be.
I tried lots of different values, yet it looks like video isn't compressed at all, same file size at the end regardless of the parameters I chose. :/

**Comment by haibert** [comment]
@pierroo nope, yea I tried a lot of different combinations as well, the app either crashes or no compression happens at all. Glad to see other people are having this issue too, I was starting to feel super dumb lol. 

I would love to just put compression to auto and move on, less work for me but it compresses it too much

**Comment by Hinjeniero** [comment]
Did you guys fix the issue? Still having it and it is pretty annoying since I require different levels of compressions

**Comment by MxBM** [comment]
Anyone encounter anything new since?

**Comment by bqrichards** [comment]
I'm not having an issue compressing video manually. This is the code I'm currently using:

```
const compressedVideoUri = await Video.compress(
  videoPath,
  {
    minimumFileSizeForCompress: 0, // minimumFileSizeForCompress is in Mb
    bitrate: 16,
  }
)
```

**Comment by v-bondarenko364** [comment]
I've tried different combinations for bitrate in my IOS simulator and it looks like it doesn't work correctly - because I write 1 minute video then compress with varying options of bitrate (16, 20, 30, 5, 1) and my output video file size is doesn't change widely + bitrate also varies slightly. In my case, bitrate and video size drastically changed only in case if I set the bitrate to "1"

**Comment by v-bondarenko364** [comment]
Here's the related issue of this library that indicates that bitrate is not doing anything under the hood - https://github.com/Shobbak/react-native-compressor/issues/108

**Comment by numandev1** [comment]
🙏 sorry for a long ago I was not maintaining due to the huge office workload, hopefully, I am going to start fixing bugs on a weekend basis,
I will fix at least 2 issues on the weekend, for priority bugs you can join the discord channel https://discord.gg/dtYzk8sp where we can discuss which bug is most important

**Comment by numandev1** [comment]
released in `1.7.2`
